### DISCUSSION TITLE
Disabling variant on multi-site disabled for all sites

### DISCUSSION THREAD

**Discussion by engram-design** [craftcms/commerce#2375] [Ideas]
Not sure if this is more a multi-store (lack thereof) issue or not. We have a multi-site setup with Commerce, and we'd like to control the variants for each. Some sites will have variants enabled, others disabled.

Disabling a variant will disable it for all sites, which it shouldn't.

Other attributes seem to be site-aware, such as the title field (but not price, and SKU - that's for multi-store).

Is this correct behaviour?

Commerce 3.1.1
Craft CMS 3.4.14

**Comment by lukeholder** [comment]
It is correct behaviour as of right now. 

Right now it acts the same way as a matrix block set to propagate to all sites - there is no per site options stored for variants like enabled/price/sku etc.

When we introduce multi-store and full multi-site support we can then do the work to get this all working as expected.
